src/Raven.Server/Config/Categories/LogsConfiguration.cs

```diff
@@ -25,5 +25,12 @@ public class LogsConfiguration : ConfigurationCategory
         [SizeUnit(SizeUnit.Megabytes)]
         [ConfigurationEntry("Logs.MaxFileSizeInMb", ConfigurationEntryScope.ServerWideOnly)]
         public Size MaxFileSize { get; set; }
+
+        [Description("How far back we should retain log entries")]
+        [DefaultValue(3 * 24)]
+        [TimeUnit(TimeUnit.Hours)]
+        [ConfigurationEntry("Logs.RetentionTimeInHrs", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
Given that this is new configuration, why do we have `Hrs` here? Do we usually have both `Hrs` and `Hours`?

Reply:
We are using _shortcuts_ (ms, sec, min, hrs) on TimeSetting fields in most configurations. Should I stick to it? or use _Hours_ ?

Reply:
`Hrs`

Reply:
done